recipes/autoray/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "autoray" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dc60095fbcc464e1d72ae7c4447438cff947e0cdb14626045b70abea8b58fe9e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - numpy
+    - python >=3.5
+
+test:
+  imports:
+    - autoray
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jcmgray/autoray/
+  summary: Write backend agnostic numeric code compatible with any numpy-ish array library.
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
This needs a licence_family entry

Reply:
Thanks, I've added `license_family: Apache`.